swagger-config.yaml

```diff
@@ -201,9 +201,54 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/StatesResult'
+  /trends:
+    get:
+      tags:
+        - Trends
+      summary: "List trends"
+      description: "Return specific aggregations for a search"
+      parameters:
+        - $ref: '#/components/parameters/search_term'
+        - $ref: '#/components/parameters/field'
+        - $ref: '#/components/parameters/size'
+        - $ref: '#/components/parameters/company'
+        - $ref: '#/components/parameters/company_public_response'
+        - $ref: '#/components/parameters/company_received_max'
+        - $ref: '#/components/parameters/company_received_min'
+        - $ref: '#/components/parameters/company_response'
+        - $ref: '#/components/parameters/consumer_consent_provided'
+        - $ref: '#/components/parameters/consumer_disputed'
+        - $ref: '#/components/parameters/date_received_max'
+        - $ref: '#/components/parameters/date_received_min'
+        - $ref: '#/components/parameters/has_narrative'
+        - $ref: '#/components/parameters/issue'
+        - $ref: '#/components/parameters/product'
+        - $ref: '#/components/parameters/state'
+        - $ref: '#/components/parameters/submitted_via'
+        - $ref: '#/components/parameters/tag'
```

Comment:
`tags` not `tag`

Reply:
Good catch - that was wrong on most of the endpoints. Fixed it